package.json

```diff
@@ -2,5 +2,8 @@
   "type": "module",
   "scripts": {
     "extract-schemas": "node scripts/extract-schemas.js"
+  },
+  "dependencies": {
```

Comment:
I don't think so, but that's only because I'm not aware of any use cases where other npm modules would depend on this one (since that's the primary use case for the `dependencies`/`devDependencies` divide). In fact, I was expecting to help prevent accidental publication of this module by adding [`"private": true`](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#private) to the manifest.

Maybe I'm overlooking something, though! Does my explanation suggest anything to that effect?

Reply:
makes sense! No action needed here